docs/guides/io_config.mdx

```diff
@@ -31,23 +37,39 @@ default:
     type: service_account
     project_id: project-id
     private_key_id: key-id
-    private_key:
-      "-----BEGIN PRIVATE KEY-----\nyour_private_key\n-----END_PRIVATE_KEY"
+    private_key: "-----BEGIN PRIVATE KEY-----\nyour_private_key\n-----END_PRIVATE_KEY"
     client_email: your_service_account_email
     auth_uri: "https://accounts.google.com/o/oauth2/auth"
     token_uri: "https://accounts.google.com/o/oauth2/token"
     auth_provider_x509_cert_url: "https://www.googleapis.com/oauth2/v1/certs"
     client_x509_cert_url: "https://www.googleapis.com/robot/v1/metadata/x509/your_service_account_email"
   GOOGLE_SERVICE_ACC_KEY_FILEPATH: "/path/to/your/service/account/key.json"
+  GOOGLE_LOCATION: US # Optional
+  MONGODB_DATABASE: database
+  MONGODB_HOST: host
+  MONGODB_PASSWORD: password
+  MONGODB_PORT: 27017
+  MONGODB_COLLECTION: collection
+  MONGODB_USER: user
+  MSSQL_DATABASE: database
+  MSSQL_SCHEMA: schema
+  MSSQL_DRIVER: "ODBC Driver 18 for SQL Server"
+  MSSQL_HOST: host
+  MSSQL_PASSWORD: password
+  MSSQL_PORT: 1433
+  MSSQL_USER: SA
   MYSQL_DATABASE: database
   MYSQL_HOST: host
   MYSQL_PASSWORD: password
   MYSQL_PORT: 3306
   MYSQL_USER: root
+  POSTGRES_CONNECT_TIMEOUT: 10
   POSTGRES_DBNAME: postgres
+  POSTGRES_SCHEMA: public # Optional
   POSTGRES_USER: username
   POSTGRES_PASSWORD: password
   POSTGRES_HOST: hostname
+  REDSHIFT_SCHEMA: public # Optional
```

Comment:
move this below. it should be together with the redshift config fields.

Reply:
Good catch: updated.